source/404.md

```diff
@@ -6,13 +6,16 @@ title: 404 Page Not Found
 
 # 404 Page Not Found
 
-The page was moved or never existed. Use menu above to navigate.
+The page was moved or never existed.  
+Use menu above to navigate or go to [home page](/).  
+If you noticed any site problem, typos, etc on any page, 
+please correct using "Edit on GitHub" button or check on <https://github.com/apache/struts-site>.
 
 You will be redirected to main page in 5 second.
```

Comment:
I think it should be `5 seconds`. And now with this new page, 5 seconds is not enough to read all I think 😉

Reply:
do it 10 sec.

for me 5 sec is also not enough.

Reply:
BTW, it does not redirect any more, if user uses browser Back button.

Reply:
I proposed a fix by #77

Reply:
The `Edit on GitHub` removed because the above `Fork me on GitHub` forks Struts itself rather than the site.

Reply:
I tested my browser (Chrome) back button and your redirect works here :)

Reply:
> The Edit on GitHub removed because the above Fork me on GitHub forks Struts itself rather than the site.

Edit on GitHub is button on the left defined in `default` layout.  And it is for the https://github.com/apache/struts-site  

While "Fork me on GitHub" is image put on page right side.

Reply:
I use Firefox. Chrome is not good without access to Chrome store plugins.

And any google services are not available in China. (And I can't use VPN on this computer)

Reply:
Thanks! You are right. I didn't see that button at home and 404 page and forgot that button. Sorry :(

Reply:
Do you get any error appeared in your browser console when you back button to 404?

Reply:
No. I want to believe that Firefox is smart to prevent redirect, when i asked to return.

Reply:
[That's correct and that's a good thing.](https://stackoverflow.com/a/2639165/1362623)